src/components/pages/Stats/Totals/StatsTotalSection.tsx

```diff
@@ -47,13 +60,19 @@ const StatsTotalSection = ({ setNetDeposits, setNetDebt }) => {
   const hasDepositsInPool2 = !!balance?.SLP;
 
   // Tranches
-  const daiSPrincipal = usePrincipal(TranchePool.DAI, TrancheRating.S);
-  const daiAPrincipal = usePrincipal(TranchePool.DAI, TrancheRating.A);
-  const totalPrincipal = usePrincipalBalance();
+  const daiSPrincipal: string | undefined = usePrincipal(
+    TranchePool.DAI,
+    TrancheRating.S
+  );
+  const daiAPrincipal: string | undefined = usePrincipal(
+    TranchePool.DAI,
+    TrancheRating.A
+  );
+  const totalPrincipal: string | undefined = usePrincipalBalance();
```

Comment:
Can put these typings at the hook level instead of here

Reply:
Nvm looks like they are typed there too. Just always make sure typings are present on the soursce of the data not just the consumer!